index.js

```diff
@@ -68,7 +68,52 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+
+header.addEventListener('click', function() {
+  toggleColor(header)
+})
+
+// header.addEventListener('click', toggleColor)
+// why doesn't this work?
+
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
+
+// likeBtn.addEventListener('click', function(){
+//   const likes = parseInt(pLike.textContent) + 1 + " Likes"
+//   pLike.textContent = likes
+// })
+
+likeBtn.addEventListener('click', function(){
+  const likes = parseInt(pLike.textContent)
+  pLike.textContent = likes + 1 + " Likes"
+})
 
 /***** Deliverable 3 *****/
+
+//link https://youtu.be/6rmUwoM-KQs?t=131
+
+const form = document.querySelector('#new-goal-form')
+const goalLinkForm = form.querySelector('#goal-link')
+const goalDescription = form.querySelector('#goal-description')
+
+form.addEventListener('submit', function(event){
+  event.preventDefault()
+
+  const link = event.target.link.value
+  const description = event.target.description.value
+  
+  const goalId = document.querySelector('#goals').lastChild.dataset.id
+  const newId = parseInt(goalId) + 1
+
+  const newGoal = {
+    id: newId,
+    link: link,
+    description: description
+  }
```

Comment:
Well done! I like your solution for getting the goal ID. Once we get to working with fetch, the server will take care of making the ID for us. If you want to save on variables declarations, you could also create your object like so:

```js
  const newGoal = {
    id: newId,
    link: event.target.link.value,
    description: event.target.description.value
  }
```